stock_picking_show_return/i18n/es.po

```diff
@@ -1,45 +1,56 @@
 # Translation of Odoo Server.
 # This file contains the translation of the following modules:
-# * stock_picking_show_return
+# 	* stock_picking_show_return
 #
-# Translators:
-# OCA Transbot <transbot@odoo-community.org>, 2017
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 12.0c\n"
+"Project-Id-Version: Odoo Server 14.0+e\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-01-30 20:51+0000\n"
-"PO-Revision-Date: 2020-01-30 21:52+0100\n"
-"Last-Translator: OCA Transbot <transbot@odoo-community.org>, 2017\n"
-"Language-Team: Spanish (https://www.transifex.com/oca/teams/23907/es/)\n"
-"Language: es\n"
+"POT-Creation-Date: 2021-08-17 03:17+0000\n"
+"PO-Revision-Date: 2021-08-17 03:17+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
-"Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Generator: Poedit 2.0.6\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
 
 #. module: stock_picking_show_return
 #: model_terms:ir.ui.view,arch_db:stock_picking_show_return.view_picking_form
 msgid "<span class=\"o_stat_text\">Source picking</span>"
-msgstr "<span class=\"o_stat_text\">Albarán origen</span>"
+msgstr ""
+
+#. module: stock_picking_show_return
+#: model:ir.model.fields,field_description:stock_picking_show_return.field_stock_picking__display_name
+msgid "Display Name"
+msgstr ""
+
+#. module: stock_picking_show_return
+#: model:ir.model.fields,field_description:stock_picking_show_return.field_stock_picking__id
+msgid "ID"
+msgstr ""
+
+#. module: stock_picking_show_return
+#: model:ir.model.fields,field_description:stock_picking_show_return.field_stock_picking____last_update
+msgid "Last Modified on"
+msgstr ""
 
 #. module: stock_picking_show_return
 #: model:ir.model.fields,field_description:stock_picking_show_return.field_stock_picking__returned_ids
 msgid "Returned pickings"
-msgstr "Albaranes devueltos"
+msgstr ""
```

Comment:
Is it intended to drop these translations during the migration?

Reply:
No, it's not, solved